packages/p2p/src/components/order-details/order-details-confirm-modal.jsx

```diff
@@ -13,44 +12,17 @@ const OrderDetailsConfirmModal = ({
     hideConfirmOrderModal,
     should_show_confirm_modal,
 }) => {
-    const {
-        amount,
-        account_currency,
-        amount_display,
-        id,
-        local_currency,
-        other_user_details,
-        // price_display,  TODO: Uncomment when price is fixed
-        rate,
-    } = order_information;
+    const { account_currency, amount_display, local_currency, other_user_details, price_display } = order_information;
 
-    const isMounted = useIsMounted();
-    const [error_message, setErrorMessage] = React.useState('');
-    const [is_checkbox_checked, setIsCheckboxChecked] = React.useState(false);
-    const [is_process_request, setIsProcessRequest] = React.useState(false); // This state disables the Release amount button during a request
+    const { order_details_store, order_store } = useStores();
 
-    const confirmOrderRequest = () => {
-        setIsProcessRequest(true);
-        requestWS({
-            p2p_order_confirm: 1,
-            id,
-        })
-            .then(response => {
-                if (isMounted()) {
-                    if (response.error) {
-                        setErrorMessage(response.error.message);
-                    }
-                }
-            })
-            .finally(() => setIsProcessRequest(false));
-    };
+    const [is_checkbox_checked, setIsCheckboxChecked] = React.useState(false);
 
     const getConfirmButtonText = () => {
         if (is_buy_order_for_user) {
             return <Localize i18n_default_text="I've paid" />;
```

Comment:
Do we need an extra variable for the confirm button text? Since `getConfirmButtonText` is not reused anywhere else and we are also initializing the `Localize` component within the parent component

Reply:
Since I'll have to use a ternary it's okay to keep it as a function and also okay to use a ternary in the component